src/extensions/default/InlineTimingFunctionEditor/main.less

```diff
@@ -379,3 +379,5 @@
         }        
     }
 }
+
+@sansFontFamily: Helvetica, Arial, "Meiryo UI", "ＭＳ Ｐゴシック", "MS PGothic", sans-serif;
```

Comment:
Can you confirm that defining variables after they are used is OK with LESS?  I'm not sure, but you might want to move this to the top of the file.  I might be wrong, check it out?